service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadDocumentsController.java

```diff
@@ -66,6 +66,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             data.put("applicationDocumentsToFollowReason",
                 migratedData.getOrDefault("applicationDocumentsToFollowReason", null)
             );
+            data.put("courtBundleList", migratedData.getOrDefault("courtBundleList", null));
```

Comment:
can we do `caseDetails.getData().putAll(uploadDocumentsMigrationService.transformFromOldCaseData(caseData));` instead of data.put.. data.put... data.put..., and handle the null logic in the service i.e `        data.put("courtBundleList", !courtBundleList.isEmpty() ? courtBundleList : null);
`
I missed this because of how we are handling these maps

Reply:
👍